<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RLLibVizMediator</class>
 <widget class="QMainWindow" name="RLLibVizMediator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RLLibViz</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="mainWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>791</width>
      <height>641</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>16</width>
      <height>721</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>1</y>
      <width>427</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>RLLib Demos</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonSwingPendulumOffPAC">
           <property name="text">
            <string>SwingPendulum (OffPAC)</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QRadioButton" name="radioButtonContinuousGridworldOffPAC">
           <property name="text">
            <string>ContinuousGridworld (OffPAC)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="radioButtonMountainCarOffPAC">
           <property name="text">
            <string>MountainCar (OffPAC)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButtonSwingPendulumAverageRewardActorCritic">
           <property name="text">
            <string>SwingPendulum (AverageRewardActorCritic)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QRadioButton" name="radioButtonMountainCarSarsaAlphaBound">
           <property name="text">
            <string>MountainCar (SarsaAlphaBound) {Random}</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButtonSwingPendulumSarsaTrue">
           <property name="text">
            <string>SwingPendulum (SarsaTrue)</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QRadioButton" name="radioButtonMountainCarSarsaAlphaBoundFourierBasis">
           <property name="text">
            <string>MountainCar (SarsaAlphaBound) {FourierBasis|Random}</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButtonSwingPendulumSarsaAlphaBoundFourierBasis">
           <property name="text">
            <string>SwingPendulum (SarsaAlphaBound) {FourierBasis}</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QRadioButton" name="radioButtonCartPoleSarsaAlphaBoundFourierBasis">
           <property name="text">
            <string>CartPole (SarsaAlphaBound) {FourierBasis}</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QRadioButton" name="radioButtonAcrobotSarasaAlphaBoundFourierBasis">
           <property name="text">
            <string>Acrobot (SarsaAlphaBound) {FourierBasis}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Simulation Speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderSimulationSpeed">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarSimulationSpeed">
       <property name="value">
        <number>99</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Instructions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>1. Select a demo from &quot;RLLib Demos&quot;.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>2. Press &quot;Exec&quot; button to excute the demo.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>3. Press &quot;Stop&quot; to stop execution of the demo. </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>4. Use &quot;Simulation Speed&quot; slider to change the</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>    simulation speed.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonExec">
         <property name="text">
          <string>Exec</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>mainWidget</zorder>
   <zorder>line</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
